vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboard.html

```diff
@@ -1,9 +1,49 @@
 <html layout:decorate="~{layouts/main_staff}">
 <head>
-	<script>
+	<script th:inline="javascript" >
+		var admin = [[${#authorization.expression("hasRole('ROLE_ADMIN')")}]];
+		var spaceLinksSize = [[${spaceLinksSize}]];
 		$(function () {
 		  $('[data-toggle="tooltip"]').tooltip()
 		});
+		
+	function closeSpaceLinkAlert(){
+		$("#spaceLinkAlert").hide();
+	}
+	
+	function closeSpaceLinksAlert() {
+		$("#spaceLinksAlertDeletionMessage").hide();
+	}
+	
+	function close(){
+		$("#error").hide();
+	}
+	
+	function deleteSpaceLinks(){
+		var token =  $('input[name="_csrf"]').attr('value');
+		var url = "[(@{'/staff/dashboard/cleanup'})]";
+		$.ajax({
+    		url: url,
+    		type: 'DELETE',
+    		headers: {
+                'X-CSRF-Token': token 
+           	},
+    		success: function(data) {
+    			$("#spaceLinkAlert").hide();
+    			$("#spaceLinksAlertDeletionMessage").show();
+    		},
+    		error: function(data) {
+    			$("#spaceLinkAlert").hide();
+    			$("#error").show();
+    		}
+    	});
+	}
+	
+	$(window).on('load', function () {
+    	if(admin == true && spaceLinksSize != null && spaceLinksSize > 0){
+    		$("#spaceLinkAlert").show();
+    	}
```

Comment:
have you tried this instead:
```
/*[# sec:authorize="hasAnyRole('ROLE_ADMIN')"]*/
    $(window).on('load', function () {
    	if(spaceLinksSize != null && spaceLinksSize > 0){
    		$("#spaceLinkAlert").show();
    	}
/*[/]*/
```

Reply:
Yes this is working thanks, I thought `/* .. */` indicated that we were commenting it out, and I was trying it without that.
Now it works. Raising a PR